WcaOnRails/app/models/team.rb

```diff
@@ -56,6 +57,7 @@ def self.all_councils
     ]
   end
 
+  # Don't forget that the WFC Leader is an officer too, the WCA Treasurer!
```

Comment:
This comment doesn't quite belong on this PR, but I felt it was better than sending in a PR just for it :stuck_out_tongue: